src/geometries/polytopes/pyramid.jl

```diff
@@ -6,11 +6,18 @@
     Pyramid(p1, p2, p3, p4, p5)
 
 A pyramid with points `p1`, `p2`, `p3`, `p4`, `p5`.
+
+The first four points form the base and the last is the is the apex.
```

Comment:
```suggestion
The first four points form the base and the last is the apex.
```
We are getting there... 😆